### ISSUE TITLE
How to use the training dataset?

### ISSUE THREAD

**Issue by erlilyu** [yilundu/reduce_reuse_recycle#4]
Hi, I'm trying to train the model. However, the training data (https://www.dropbox.com/scl/fi/b8stn55nalcdegdp0aj5y/dataset_train_larger.npz?rlkey=t0g1baca73lluaxp19icgqgo1&st=ibhozr5x&dl=0) provided cannot be used to train the model.

I find in the training file that the dataset is set as "clevr_norel". It seems to me that "clevr_norel" should have one dimension label for each image. However, the provided dataset has two dimension labels. That causes a mismatch in line 1356 of file "unet.py".

Thanks for your attention. I look forward to your response.

Regards,
Erli

**Comment by yilundu** [comment]
Hi Erli,

You will probably need to modify the model so that it can properly load the data. An undergrad
researcher wrote the code, so there is a bit of a mismatch between data / model

Best,
Yilun

**Comment by erlilyu** [comment]
Hi Yilun,

Thank you for your response.

The current situation is totally understandable. At this point, I'd like to try modifying the model to make it work. However, I'm completely unfamiliar with the CLEVR dataset, especially the meaning of the labels. Do you have any suggestions or hints?

Regards,
Erli

**Comment by yilundu** [comment]
Hey! You can potentially take a look at the code here: https://github.com/energy-based-model/Compositional-Visual-Generation-with-Composable-Diffusion-Models-PyTorch

This might be helpful in loading the CLEVR dataset.

**Comment by erlilyu** [comment]
Noted, thanks
